adapta/storage/delta_lake/v2/_functions.py

```diff
@@ -42,6 +42,7 @@ def load(  # pylint: disable=R0913
     auth_client: AuthenticationClient,
     path: DataPath,
     version: Optional[int] = None,
+    timestamp: Optional[Union[str, datetime]] = None,
```

Comment:
I would probably just force the user to use datetimes

Reply:
good idea